### PR TITLE
Explode aliases

### PR DESCRIPTION
**Pull request by darkpixel**
Changes proposed in this pull request:
- Allow aliases to be 'exploded' to an array of recipients in addition to a single recipient

### CODE REVIEW

**Comment by darkpixel** [comment]
Stupid spacing.  That's what I get for making changes against a remote box instead of my local copy of vim with its properly configured eslint...

File: plugins/aliases.js [hunk 1]
```diff
@@ -86,21 +86,28 @@ function _alias(plugin, connection, key, config, host) {
...
-        if (config.to.search("@") !== -1) {
-            to = config.to;
+        if (Array.isArray(config.to)) {
+            connection.logdebug(plugin, "aliasing " + connection.transaction.rcpt_to + " to " + config.to);
+            connection.transaction.rcpt_to.pop();
+            for (var i = 0, len = config.to.length; i < len; i++) {
+                toAddress = new Address('<' + config.to[i] + '>');
+                connection.transaction.rcpt_to.push(toAddress);
+            }
+        } else {
```
Comment (baudehlo) [inline plugins/aliases.js]: This makes me stabby :)

File: plugins/aliases.js [hunk 2]
```diff
@@ -86,19 +86,28 @@ function _alias(plugin, connection, key, config, host) {
...
+            for (var i = 0, len = config.to.length; i < len; i++) {
+                toAddress = new Address('<' + config.to[i] + '>');
+                connection.transaction.rcpt_to.push(toAddress);
+            }
         }
         else {
-            to = config.to + '@' + host;
+            if (config.to.search("@") !== -1) {
+                to = config.to;
+            } else {
```
Comment (baudehlo) [inline plugins/aliases.js]: You missed one.

**Comment by baudehlo** [comment]
aliases already has a test file. Please add tests for this to it.

**Pull request merged**